[quant][graphmode] Skip observing bias across function call hierarchy

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #26696 [refactor] move more functions to InsertObserversHelper
* **#26642 [quant][graphmode] Skip observing bias across function call hierarchy**
* #26641 [refactor] Add ObserveHelper and remove some common function parameters
* #26640 [quant][graphmode][fix] remove unneeded code

Summary:
att

Test Plan:
python test/test_jit.py 'TestJit.test_insert_observers'

Reviewers:
mvz
Subscribers:

Tasks:

Tags:

Differential Revision: [D17538667](https://our.internmc.facebook.com/intern/diff/D17538667)

jerryzh168 committed:
[quant][graphmode] Skip observing bias across function call hierarchy

Summary:
att

Test Plan:
python test/test_jit.py 'TestJit.test_insert_observers'

Reviewers:
mvz
Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM commented on torch/csrc/jit/passes/quantization.cpp:
```diff
@@ -182,12 +182,13 @@ class ObserveHelper {
       script::Module& module,
       const QConfig& qconfig);
-  void propagateWeightValues(Node* n, std::shared_ptr<Graph>& graph);
+  void propagateValues(Node* n, std::shared_ptr<Graph>& graph);
```
This name became very generic - please add a comment what exactly is getting propagated and where.

ZolotukhinM reviewed (approved):
👍

jerryzh168 committed:
Update on "[quant][graphmode] Skip observing bias across function call hierarchy"


Summary:
att

Test Plan:
python test/test_jit.py 'TestJit.test_insert_observers'

Reviewers:
mvz
Subscribers:

Tasks:

Tags:

[ghstack-poisoned]
